Mathlib/Topology/Covering/AddCircle.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2025 Junyan Xu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Junyan Xu
+-/
+import Mathlib.Topology.Instances.AddCircle
+import Mathlib.Topology.Covering.Galois
+
+/-!
+# Basic examples of Galois covering maps
+
+-/
+
+theorem isCoveringMap_coe_addCircle (p : ℝ) : IsCoveringMap ((↑) : ℝ → AddCircle p) :=
+  AddSubgroup.isCoveringMap _
```

Comment:
Could you also add a version for `expMapCircle`?

Reply:
Also, does this generalize to the same generality as `isLocalHomeomorph_coe`?

Reply:
Actually
```
variable {𝕜 : Type*} [AddCommGroup 𝕜] (p : 𝕜) [LinearOrder 𝕜] [IsOrderedAddMonoid 𝕜]
  [TopologicalSpace 𝕜] [OrderTopology 𝕜] [DiscreteTopology (AddSubgroup.zmultiples p)]
```
suffice and `0 < p`, `[Archimedean 𝕜]` and ` [DenselyOrdered 𝕜]` are not needed.
`[LinearOrder 𝕜] [IsOrderedAddMonoid 𝕜] [OrderTopology 𝕜]` can also be replaced by `[IsTopologicalAddGroup 𝕜]` (which would make this theorem a direct application of `AddSubgroup.isCoveringMap`.

Reply:
Yeah, we can use [IsCoveringMap.isLocalHomeomorph](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Covering.html#IsCoveringMap.isLocalHomeomorph) and it would only add ~6 imports on top of ~1220. I guess `isLocalHomeomorph` isn't true under weaker conditions than `isCoveringMap`, so that might be it.

Reply:
For `expMapCircle`, I've proven that `exp : C -> C \ {0}` is a covering map [here](https://github.com/leanprover-community/mathlib4/commit/c224539377c581cbf9fa1373268e44d2f91a4bb7#diff-a676a5b63486aa7ef960d2f8f1da7905e0094b28507f44322ccbd6009d34dc6cR17) and I'm also able to restrict a covering to a subspace, but I still need pre/post-composing covering maps with homeomorphisms.